graylog2-server/src/main/java/org/graylog/security/authservice/backend/ADAuthServiceBackendConfig.java

```diff
@@ -104,8 +105,8 @@ public void validate(ValidationResult result) {
     }
 
     @Override
-    public List<String> hostAllowList() {
-        return servers().stream().map(HostAndPort::host).toList();
+    public Optional<List<String>> externalHTTPHosts() {
+        return Optional.of(servers().stream().map(HostAndPort::toString).toList());
```

Comment:
Active Directory is just a slightly different LDAP implementation. So we should return `Optional.empty()` here.

Reply:
done